certified-connectors/Data8/apiDefinition.swagger.json

```diff
@@ -1,362 +1,818 @@
+                      "title": "Credits Remaining"
+                    }
                   },
-                  "ShortBankName": {
-                    "type": "string",
-                    "description": "A short name for the bank indicated by the sort code.",
-                    "title": "Short Bank Name"
+                  "description": "Status"
+                },
+                "Result": {
+                  "type": "object",
+                  "properties": {
+                    "Address": {
+                      "type": "object",
+                      "properties": {
+                        "Lines": {
+                          "type": "array",
+                          "items": {
+                            "type": "string",
+                            "title": "Address Lines"
+                          },
+                          "description": "Lines"
+                        }
+                      },
+                      "description": "Address",
+                      "title": "Address Lines"
+                    }
                   },
-                  "FullBankName": {
-                    "type": "string",
-                    "description": "The full name for the bank indicated by the sort code.",
-                    "title": "Full Bank Name"
+                  "description": "Result"
+                },
+                "MatchLevel": {
+                  "type": "string",
+                  "description": "Contains a MatchLevel to indicate how well the address matched to PAF. Possible values are OK_Unchanged, OK_Corrected, NOK_Building, NOK_Street, NOK_Town, NOK_NoMatch",
+                  "title": "Match Level"
+                },
+                "CountryName": {
+                  "type": "string",
+                  "description": "Contains the name of the country the address is in.",
+                  "title": "Country Name"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/AddressCapture/GetFullAddress.json": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Status": {
+                  "type": "object",
+                  "properties": {
+                    "Success": {
+                      "type": "boolean",
+                      "description": "Success"
+                    },
+                    "ErrorMessage": {
+                      "type": "string",
+                      "description": "Error Message (if applicable)"
+                    },
+                    "CreditsRemaining": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "Credits Remaining",
+                      "title": ""
```

Comment:
@mattybeard Empty property values can be removed until unless you have a specific reason to keep them.